packages/autorest.typescript/test/rlcIntegration/generated/bodyFileRest/package.json

```diff
@@ -27,11 +27,11 @@
     "build:samples": "echo skipped.",
     "build:test": "echo skipped.",
     "build:debug": "echo skipped.",
-    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\"  \"samples-dev/**/*.ts\"",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\" ",
     "clean": "rimraf dist dist-browser dist-esm test-dist temp types *.tgz *.log",
     "execute:samples": "echo skipped",
     "extract-api": "rimraf review && mkdirp ./review && api-extractor run --local",
-    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\"  \"samples-dev/**/*.ts\"",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\" ",
```

Comment:
Small fix when there is no sample folder included.

Reply:
This is the small fix, sample folder should be ignored if there is no samples generated.

Should not be ignored if there is samples generated.
https://github.com/Azure/autorest.typescript/blob/2b5698fb00ab84a2469c30ee19c9451c40e18695/packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/package.json#L34

Reply:
can you point to me where you add this check in the codegen side ?

Reply:
https://github.com/Azure/autorest.typescript/pull/1967/files#diff-c986168a4919c6df1a665149746b826e4876f0a438955d39d098852eec9abc53R74-R76